install_config/certificate_customization.adoc

```diff
@@ -284,6 +284,64 @@ openshift_hosted_registry_routetermination=reencrypt <3>
 +
 * Set to `passthrough` to terminate encryption at the destination. The destination is responsible for decrypting traffic.
 
+[[configuring-custom-certificates-lb]]
+== Configuring a Custom Certificate for a Load Balancer
+
+If your {product-title} cluster uses the default load balancer or an enterprise-level load balancer, 
+you can use custom certificates to make the web console and API available externally using a publicly-signed custom certificate, leaving the existing internal certificates for 
```

Comment:
s/using/by using

That's really long. Maybe s/, leaving the existing internal certificates for the internal endpoints./. The internal endpoints still use the existing internal certificates.

Reply:
I am tempted to leave this as is; as that statement was exactly what the customer is asking for.